### ISSUE TITLE
Duda

### ISSUE THREAD

**Issue by jmaurici** [SergioGMR/esports#1]
Hola Sergio.

Este repositorio lo vas a usar para el proyecto Java?

Saludos
JMauricio

**Comment by SergioGMR** [comment]
sí, estaba haciendo el commit inicial en la rama dev, para ir desarrollando ahí y cuando vaya teniendo algo que valga la pena pasandolo a master.
